easybuild/easyblocks/o/openfoam.py

```diff
@@ -389,12 +389,19 @@ def install_step(self):
             libdir = os.path.join(self.installdir, self.openfoamdir, "lib", psubdir)
         else:
             libdir = os.path.join(self.installdir, self.openfoamdir, "platforms", psubdir, "lib")
-        mpilibsdir = os.path.join(libdir, "mpi")
+
+        # OpenFOAM v2012 puts mpi into eb-mpi
+        if self.looseversion >= LooseVersion("2012"):
+            mpidir = "eb-mpi"
```

Comment:
`mpilibdir` maybe?

Reply:
I would prefer mpilibsubdir or mpisubdir rather than mpilibdir.  Mpilibdir is very similar to mpilibsdir (only one letter) and it may be slightly confusing. What do you think?

Reply:
No strong opinion either way